wagl/__init__.py

```diff
@@ -1,6 +1,2 @@
 from __future__ import print_function
 from __future__ import absolute_import
-from ._version import get_versions
-
-__version__ = get_versions()["version"]
```

Comment:
Any reason not to still include a `__version__` field for people? That's where I usually expect to see it, as with other projects: eg. `rasterio.__version__`, `requests.__version__`

Reply:
Other than being ugly, not really.

Just trying to avoid any potential circular dependency occurring at runtime.  setuptools_scm will create a `<module>/version.py` file which doesn't need to be committed to the repo.

I can do something like:

```python
from importlib.metadata import version
__version__ = version("wagl")
del version
```

But it does raise another point that I missed.  I use `importlib.metadata` package, which is available in 3.8, but the pre 3.8 should use `importlib_metadata`.

Alternately, I can use `from pkg_resources import get_distribution` which is older.

Reply:
Thoughts?  Would it be better to include __version__ field?